azavea/pfb-network-connectivity
# Feature/awf/bike infra fix track

## Overview

Fixes up the drawing order of the bike infrastructure lines. There were rendering issues with the different layers, specifically that the "tracks" would render as a thick green line instead of a thick teal line. 

### Demo

#### Before

![screen shot 2017-07-17 at 09 37 50](https://user-images.githubusercontent.com/1818302/28270729-85017f9a-6ad4-11e7-8607-c57b2c22c280.png)

#### After

The thick green line on the right side of the screenshot above becomes teal below:

![screen shot 2017-07-17 at 09 37 17](https://user-images.githubusercontent.com/1818302/28270753-9bfeb3e8-6ad4-11e7-91cd-1b0fd1336ebb.png)

Verification that the crosswalk filtering is correct (its difficult to tell on the FoCo shapefile):


![screen shot 2017-07-17 at 09 36 26](https://user-images.githubusercontent.com/1818302/28270799-bb92e760-6ad4-11e7-8c2c-97f3a9b78039.png)
to
![screen shot 2017-07-17 at 09 36 14](https://user-images.githubusercontent.com/1818302/28270798-bb92c7e4-6ad4-11e7-8c5c-d2c137cbddb2.png)




### Notes

This is now super annoying, as the styles can be easily broken by shifting the order around of various definitions in the CartoCSS, but I'm not sure how to improve on this right now. All ears on ways to improve this.

I also updated the FoCo shapefile in the tilemaker 'working_files' directory, so that it has the `XWALK` column.

## Testing Instructions

 * Ensure styles look correct in Mapbox Studio
 * Run Germantown job to ensure that the XWALK filtering still works correctly.
